xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/Transaction.java

```diff
@@ -232,7 +233,7 @@ default UnsignedInteger sequence() {
    *
    * @return An optionally-present {@link UnsignedLong}.
    * @deprecated This field will be removed in favor of {@link
-   *    org.xrpl.xrpl4j.model.client.transactions.TransactionResult#closeDate()};
+   * org.xrpl.xrpl4j.model.client.transactions.TransactionResult#closeDate()};
```

Comment:
This formatting looks off to me. Continuations in javadocs should have 2 spaces at the beginning of the line